server/src/main/java/com/cloud/configuration/ConfigurationManagerImpl.java

```diff
@@ -472,6 +472,8 @@ public class ConfigurationManagerImpl extends ManagerBase implements Configurati
     public static final ConfigKey<Boolean> ENABLE_DOMAIN_SETTINGS_FOR_CHILD_DOMAIN = new ConfigKey<Boolean>(Boolean.class, "enable.domain.settings.for.child.domain", "Advanced", "false",
             "Indicates whether the settings of parent domain should be applied for child domain. If true, the child domain will get value from parent domain if its not configured in child domain else global value is taken.",
             true, ConfigKey.Scope.Global, null);
+    public static final ConfigKey<Integer> HOST_RESERVED_MEM_MB = new ConfigKey<>("Advanced", Integer.class, HOST_RESERVED_MEM_MB_STRING, "1024",
+            "Set an upper limit for memory in megabytes which will be reserved for host and not used for VM allocation.", true);
```

Comment:
As hosts can behave differently between clusters, might the operators want to set a different value for each  cluster?

Reply:
cc: @wido @soreana

Reply:
Yes, I think so. As hardware might be different and maybe you have other strategies per cluster you might want to set this differently.

Reply:
+1.
would be good to change to a cluster setting